### PR TITLE
Proxy increase energy

### PR DESCRIPTION
**Pull request by dorin-iancu**

### CODE REVIEW

**Review by sasurobert — DISMISSED** [review]

**Review by alyn509 — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

File: locked-asset/proxy_dex/src/energy_update.rs [hunk 1]
```diff
@@ -46,9 +46,6 @@ pub trait EnergyUpdateModule:
                 .get_token_attributes(token_id, token_nonce);
             energy.update_after_unlock_any(token_amount, attributes.unlock_epoch, current_epoch);
         } else if token_id == &old_locked_token_id {
-            if self.blockchain().is_smart_contract(user) {
```
Comment (psorinionut) [inline locked-asset/proxy_dex/src/energy_update.rs:49]: Why was this removed?

**Review by psorinionut — APPROVED** [review]

**Review by CostinCarabas — APPROVED** [review]

File: locked-asset/proxy_dex/src/energy_update.rs (hunk 1 above)
Comment (dorin-iancu) [reply locked-asset/proxy_dex/src/energy_update.rs:49]: No longer needed. Now contracts can have energy.
